tests/rptest/tests/cluster_health_overview_test.py

```diff
@@ -125,3 +129,35 @@ def two_nodes_down():
         self.redpanda.start_node(second_down)
 
         self.wait_until_healthy()
+
+    @cluster(num_nodes=5, log_allow_list=FDT_LOG_ALLOW_LIST)
+    def test_cluster_health_data_disk_degraded(self):
+        # Should be healthy at start.
+        self.wait_until_healthy()
+
+        # Trigger low space on all nodes.
+        h = FullDiskHelper(self.logger, self.redpanda)
+        h.trigger_low_space()
+
+        def disk_degraded():
+            hov = self.get_health()
+
+            assert not hov['is_healthy'], hov
+            assert 'data_disk_degraded' in hov['unhealthy_reasons'], hov
+
+            expected_degraded_nodes = list(
+                range(1,
+                      len(self.redpanda.nodes) + 1))
```

Comment:
a comment here about the contents of this list would be helpful. like is it all nodes or a subset?